overlord/snapstate/snapmgr_test.go

```diff
@@ -641,7 +642,7 @@ func (s *snapmgrTestSuite) TestInstallRunThrough(c *C) {
 		},
 		{
 			op:   "copy-data",
-			name: "/snap/some-snap/11",
+			name: fmt.Sprintf("%s/some-snap/11", dirs.SnapMountDir),
```

Comment:
Hmm, how would you suggest I do two things:
- patch SnapMountDir in a distribution
- run tests successfully

Without completing this patch set? I agree it's a bit annoying (there's just a few more small patches after this one and it's all done).